Jenkinsfile

```diff
@@ -183,8 +183,8 @@ pipeline {
         }
         stage('Roberta Squad v1.1') {
           steps {
-            sh 'cd examples/nlp/question_answering && CUDA_VISIBLE_DEVICES=1 python question_answering_squad.py --no_data_cache --amp_opt_level O1 --train_file /home/TestData/nlp/squad_mini/v1.1/train-v1.1.json --eval_file /home/TestData/nlp/squad_mini/v1.1/dev-v1.1.json --work_dir outputs/squadv1_roberta --batch_size 2 --save_step_freq 500 --num_epochs 1 --lr_policy WarmupAnnealing  --lr 3e-5 --do_lower_case  --model_type roberta --pretrained_model_name roberta-base'
-            sh 'cd examples/nlp/question_answering && FSCORE=$(cat outputs/squadv1_roberta/log_globalrank-0_localrank-0.txt |  grep "f1" |tail -n 1 |egrep -o "[0-9.]+"|tail -n 1 ) && echo $FSCORE && if [ $(echo "$FSCORE > 50.0" | bc -l) -eq 1 ]; then echo "SUCCESS" && exit 0; else echo "FAILURE" && exit 1; fi'
+            sh 'cd examples/nlp/question_answering && CUDA_VISIBLE_DEVICES=1 python question_answering_squad.py --no_data_cache --amp_opt_level O1 --train_file /home/TestData/nlp/squad_mini/v1.1/train-v1.1.json --dev_file /home/TestData/nlp/squad_mini/v1.1/dev-v1.1.json --work_dir outputs/squadv1_roberta --batch_size 2 --save_step_freq 200 --max_steps 10 --loss_step_freq 2  --lr_policy WarmupAnnealing  --lr 1e-5 --do_lower_case  --model_type roberta --pretrained_model_name roberta-base'
+            sh 'cd examples/nlp/question_answering && FSCORE=$(cat outputs/squadv1_roberta/log_globalrank-0_localrank-0.txt |  grep "f1" |tail -n 1 |egrep -o "[0-9.]+"|tail -n 1 ) && echo $FSCORE && if [ $(echo "$FSCORE > 3.0" | bc -l) -eq 1 ]; then echo "SUCCESS" && exit 0; else echo "FAILURE" && exit 1; fi'
```

Comment:
This new F1 score (assuming that is what you're measuring) is really low

Reply:
yes, it just a dummy test to check if the F1 is not 0.

Reply:
If it is a dummy test, then what is the value of the test? I would argue that it is more important to have a stricter CI than it is to save CI test time.
@tkornuta-nvidia, do you have an opinion here?

Reply:
I think that testing that loss is non-zero is a L0 or unittest

Reply:
I agree that checking if F1 is not 0 has a low "information value" - at that point I would change really like to change that to forward-backward step. Then it will be L0.

Finally, I really think the whole "testing logic" (e.g. checking a condition) should be implemented **outside** of Jenkins configuration file. This looks like a hack, whereas python unittest provides tools devoted exactly for that purpose. This approach will make things harder to manage/reorganize/use in a long run...

Reply:
hey @tkornuta-nvidia @blisc, I have updated the criteria for the ci tests regarding your feedback, the new F1 thresh will be around ~10, which is higher than in step 0 which is ~4-7.
This diff is outdated